packages/react/src/components/CalendarDayGrid/CalendarDayGrid.base.tsx

```diff
@@ -225,6 +226,31 @@ function useWeekCornerStyles(props: ICalendarDayGridProps) {
     return style;
   };
 
+  const calculateBorderStyles = (
+    classNames: IProcessedStyleSet<ICalendarDayGridStyles>,
+    above: boolean,
+    below: boolean,
+    left: boolean,
+    right: boolean,
+  ): string => {
+    let style = '';
```

Comment:
Nit, at least IMO it would be cleaner to push to an array of strings and join them at the end.